### DISCUSSION TITLE
[iOS] Unwanted spaces in full screen mode using Media element.

### DISCUSSION THREAD

**Discussion by EdwardMcFlurry** [CommunityToolkit/Maui#1007] [Q&A] [answered]
Hello

I noticed something strange as it took me a while to spot it but there’s unnecessary/unwanted Space at the top BEFORE full screen…see image below and please note that I have a custom label/icon (Play) that is positioned according to the play control for iOS.

![CF7EFAE8-EF81-4908-8672-413254101A1B](https://user-images.githubusercontent.com/46613832/219621402-5cfb54ad-42e3-4b1a-9e38-8369136a0c21.jpeg)

The image below is when I ENTER full screen mode and notice the space above, it’s like the iOS controls have pushed the space out as I assume this is how it’s supposed to be in full screen mode…

![DAD8FFBF-B5F4-4FF5-AC95-7C72E04A329B](https://user-images.githubusercontent.com/46613832/219621992-9fd29647-3c39-4bae-9c1d-e63958393596.png)

The last image below is when I EXIT the full screen mode as the space above seems to have disappeared but my custom PLAY label/icon has lost sync with the play control for iOS.

![BAF93D7B-5E07-4996-B09E-EA602C9A4078](https://user-images.githubusercontent.com/46613832/219623750-dc3cb056-533c-4f51-936f-985ae8aa147e.jpeg)

so, I’m not sure if this is a bug in Media element but I need my custom labels/icons for PLAY & BUFFERING to remain where they are however, my issue is that this is something that my users will notice instantly.

@jfversluis our Media Element guru please take a look at it 🙏🏻

**Comment by vhugogarcia** [comment]
Could you share a repository, so we can test it? Please

**Comment by jfversluis** [comment, marked as answer]
You could try with this version: https://github.com/CommunityToolkit/Maui/pull/1008

Either get the resulting NuGet from that PR or wait until the next release. Hopefully that will fix this also. Otherwise, as @vhugogarcia mentions: please provide some code so we can have a look ourselves. Thanks!

**Reply by jfversluis** [reply]
@EdwardMcFlurry this is now released as part of version 1.0.2, please let me know if that fixes it.

**Reply by EdwardMcFlurry** [reply]
I'm so sorry for the late response and preparing the sample repo but @jfversluis I went ahead and upgraded the NuGet Package to 1.0.2 and can gladly confirm that indeed the 'unwanted space' was removed on iOS upon the upgrade and once again, @jfversluis thank you so much for all your effort. 🕺🏻

**Comment by CliffAgius** [comment]
Closed as answered
